modules/gapi/test/common/gapi_core_tests_inl.hpp

```diff
@@ -54,7 +54,7 @@ TEST_P(MathOpTest, MatricesAccuracyTest)
             }
             else
             {
-                sc += Scalar(1, 1, 1, 1);  // avoid zeros in divide input data
+                sc += Scalar(sc[0] == 0, sc[1] == 0, sc[2] == 0, sc[3] == 0);  // avoid zeros in divide input data
```

Comment:
What is the purpose of the comment then?

Reply:
It is a comment describing the purpose of this action
But written not so good
Changed to `avoiding zeros..`